STAT302 - Assignment 2 Q 2.16

In [1]:
#Q2.16 a
library(Stat2Data)
data(TextPrices)
str(TextPrices)

'data.frame':	30 obs. of  2 variables:
 $ Pages: int  600 91 200 400 521 315 800 800 600 488 ...
 $ Price: num  95 19.9 51.5 128.5 96 ...


In [2]:
model = lm(Price ~ Pages, data = TextPrices)
summary(model)


Call:
lm(formula = Price ~ Pages, data = TextPrices)

Residuals:
    Min      1Q  Median      3Q     Max 
-65.475 -12.324  -0.584  15.304  72.991 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept) -3.42231   10.46374  -0.327    0.746    
Pages        0.14733    0.01925   7.653 2.45e-08 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 29.76 on 28 degrees of freedom
Multiple R-squared:  0.6766,	Adjusted R-squared:  0.665 
F-statistic: 58.57 on 1 and 28 DF,  p-value: 2.452e-08


H0:beta1 = 0, Ha: beta1 /= 0 \
t = 7.653, df = 28 \
P-value = 2.452e-08 \
Reject H0 \
We have evidence that pages and price have linear relationship.

In [3]:
#Q 2.16 b
0.14733 - 2.048 * 0.01925
0.14733 + 2.048 * 0.01925

[1] 0.107906

[1] 0.186754

We have 95% confident when pages of book increase 1 page, the avg. price will increase (0.107906, 0.186754) dollar.